spec/system/forgotten_password_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe 'Confirmed user, following forgotten password process', type: :system do
+  let(:user) { create :user, :confirmed }
+  let(:token) { user.send_reset_password_instructions }
+
+  context 'when on the "Choose a password" page' do
+    before do
+      visit "/users/password/edit?reset_password_token=#{token}"
+    end
+
+    # Happy path scenario
+    context 'when correct credentials are entered' do
```

Comment:
Alternative options:
- `and new password meets criteria`
- `with sufficient length`

build more accurate specs when outputted as `documentation` format which makes business sense.